<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResearcherWizardPoolPage</class>
 <widget class="QWizardPage" name="ResearcherWizardPoolPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Summary</string>
  </property>
  <layout class="QVBoxLayout" name="poolPageLayout">
   <item>
    <spacer name="headerSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="poolIconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../bitcoin.qrc">:/images/ic_pool_active</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pool Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignVCenter">
    <widget class="QLabel" name="poolParagraphLabel">
     <property name="text">
      <string>In this mode, a pool will take care of staking research rewards for you. Your wallet can still earn standard staking rewards on your balance. You do not need a BOINC account, CPID, or beacon. Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="poolTableTopSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QTableWidget" name="poolTableWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>180</height>
      </size>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>grcpool</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Arikado Pool</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Website URL</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string notr="true">https://grcpool.com/</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string notr="true">https://grc.arikado.ru/</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="poolTableBottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="addressParagraphLabel">
     <property name="text">
      <string>As you sign up, the pool may ask for a payment address to send earnings to. Press the button below to generate an address.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addressLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="newAddressButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New &amp;Address</string>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="addressLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="indent">
        <number>16</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="copyToClipboardButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&amp;Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="startOverLabel">
     <property name="text">
      <string>Press &quot;Next&quot; when you are done.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
